src/controllers/loginController.ts

```diff
@@ -40,13 +41,10 @@ const loginController = async (req: Request, res: Response) => {
         'Please check your email to verify your account before login.'
       );
     }
-
-    // generate token
-    const token = signToken({ id: user.id });
-
-    return sendResponse<string>(res, 200, token, 'Logged In Successfully');
+    await Checkrole(user.id, email, req, res);
```

Comment:
wrong implementation

Reply:
any idea of how i can do it in another way sir?

Reply:
this should be else not expression